wagtail/contrib/modeladmin/templates/modeladmin/includes/result_count.html

```diff
@@ -0,0 +1,4 @@
+{% load i18n %}
+{% block result_count %}
+<span class="result-count">{{ result_count }} {% trans "out of" %} {{ all_count }}</span>
```

Comment:
The string "out of" is likely to be difficult for translators to translate in isolation. Let's do [what the pagination footer does](https://github.com/wagtail/wagtail/blob/90c86e76d0d6f46208eb1efb2918026d61ee3326/wagtail/admin/templates/wagtailadmin/shared/pagination_nav.html#L12)... 
```suggestion
<span class="result-count">{% blocktrans %}{{ result_count }} out of {{ all_count }}{% endblocktrans %}</span>
```